docs/components_and_bundles/bundles/SyliusMailerBundle/using_custom_adapter.rst

```diff
@@ -40,13 +40,6 @@ In your ``services.yml`` file, simply add your adapter definition.
             parent: sylius.email_sender.adapter.abstract
             class: App\Mailer\Adapter\CustomAdapter
 
-Configure The New Adapter
--------------------------
-
-Now you just need to put service name under ``sylius_mailer`` configuration in ``app/config/config.yml``.
```

Comment:
IMHO, we should mention, that it should be placed in some config file in packages folder